README.md

```diff
@@ -1 +1,149 @@
-# ReactRailBackEnd
\ No newline at end of file
+<a name="readme-top"></a>
+
+# Hello Rails Back End
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
```

Comment:
- [ ] Kindly remove the link to the `Live Demo` section as this section has been removed making the link broken